apps/dashboard/app/views/batch_connect/session_contexts/_path_selector.html.erb

```diff
@@ -2,6 +2,7 @@
   show_hidden = field_options.fetch(:show_hidden, false)
   show_files = field_options.fetch(:show_files, true)
   inital_directory = field_options.fetch(:directory, CurrentUser.home)
+  file_picker_favorites = field_options.fetch(:file_picker_favorites, OodFilesApp.new.favorite_paths)
```

Comment:
Now that I see this, I think we should likely have a `boolean` to show favorites or not, but not allow for redefinition - at least not yet. 

I'm trying to think of a use case where the user (the admin setting the app up OR the user interacting with it) would need an entry here that's not already in `OodFilesApp#favorite_paths`, but can't really think of one.

Reply:
Wouldn't this still require grabbing the filesPath in the JavaScript? If these paths don't come in with a filepath/file system, they don't work without it, and if you add `fs: 'fs'` to the `files_path` helper it would end up returning `fs/tmp`, right?

Reply:
> Wouldn't this still require grabbing the filesPath in the JavaScript?

I may have to look at it a little closer, because I do recall that we show/expose `filesPath` but need to refresh my mind as to why.

> If these paths don't come in with a filepath/file system, they don't work without it, 

Yea I think we should hard code/expect/support only local file systems. And this is by hard coding the `fs: 'fs'` argument.

> it would end up returning `fs/tmp`, right?

`files_path('/tmp', fs: 'fs')` 
would resolve to
`/pun/sys/dashboard/files/fs/tmp` (or the dev variant depending) which is the valid URL path.

The `fs` here in the URL indicates it's the local filesystem we're looking at. Any other value is a remote file system.